celery/backends/arangodb.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf-8 -*-
+"""Couchbase result store backend."""
```

Comment:
Is this a copy and paste error? (Couchbase vs. ArangoDB?)

Reply:
Yes.